airflow/hooks/http_hook.py

```diff
@@ -0,0 +1,84 @@
+import logging
+
+import requests
+
+from airflow import utils
+from airflow.hooks.base_hook import BaseHook
+
+
+class HttpHook(BaseHook):
+
+    """
+    Interact with HTTP servers.
+    """
+
+    def __init__(
+            self, method='POST', http_conn_id='http_default'):
+        self.http_conn_id = http_conn_id
+        self.method = method
+
+    def get_conn(self, headers):
```

Comment:
Shouldn't default should be None for headers
`def get_conn(self, headers=None):`